Hummingbird.docc/Tutorials/MongoKitten/Resources/code/Setup/mongokitten-setup-06.swift

```diff
@@ -0,0 +1,57 @@
+import Hummingbird
+import Logging
+import MongoKitten
+import Foundation
```

Comment:
Why do you need Foundation?

Reply:
We don't anymore, I was using a Date as part of the model at first